Makefile

```diff
@@ -102,4 +108,25 @@ clean:
 all: lint metalint test-ci-unit services build-ui
 	@echo Made all successfully
 
+.PHONY: install-licence-bin
+install-license-bin: install-vendor
+	@echo Installing node modules
```

Comment:
I think this should be
```
.PHONY: install-licence-bin
install-license-bin: install-vendor
	@echo Installing node modules
	git submodule update --init --recursive
	[ -d $(license_node_modules) ] || (cd $(license_dir) && npm install)
```
?

Reply:
O yeah. I was looking at the collector here:
https://github.com/m3db/m3collector/blob/master/Makefile#L46

But it looks like we are not consistent across m3 repos for example here:
https://github.com/m3db/m3metrics/blob/master/Makefile

I can put up a PR in the collector to fix this.